### ISSUE TITLE
12.10 and steps per e

### ISSUE THREAD

**Issue by dereed999** [daid/LegacyCura#259]
Has the wizard for computing steps per e been removed from 12.10?  I downloaded and installed 12.10 (os x) and went to re-run the initial wizard after having been tinkering with my Ultimaker and I'm somehow not finding the "steps per e" wizard.  I'm sure I'm just confused :)  Thanks!

**Comment by Filopat** [comment]
I have same issue in Windows 7.
 Also hitting "print" button does nothing.

**Comment by daid** [comment]
It's no issue, the steps-per-E calibration was hard to follow so I disabled it. It also only minor improved the printing result. You can still set the steps per E in the preferences, but right no there is no wizard for it.

The "print does nothing" is a separate issue, which I've heard from earlier today.

**Comment by dereed999** [comment]
Thank you daid!  I hope you do bring it back.  I found it useful and simple.  (I actually thought it made a big impact on printing quality!)  Your work is great.  Thank you again for all of it!

**Comment by quikshot** [comment]
Yes, I am missing it, too. I have a Geo Hagen extruder bolt and my steps-per-e are not standard. As this value is not stored (maybe could be stored if Cura was previously installed), I need to refind it at every new Cura version.
Couldn't it be added in expert section?
